### DISCUSSION TITLE
Is the download speed normally this slow?

### DISCUSSION THREAD

**Discussion by BitterSweetTreat** [n00mkrad/flowframes#334] [Q&A]
My internet speed is supposed to allow me to download the files necessary for Flowframes faster than what I've been getting since the past couple days. I've tried downloading using both available servers yet they both give the same download speed. 
![image](https://github.com/user-attachments/assets/41b52d2c-d807-46a8-91cc-37bc6ad550fe)
![Screenshot 2024-10-10 154040](https://github.com/user-attachments/assets/5b7a406d-a6ee-45a7-b65e-6da45c0ced81)

**Comment by Rizip4** [comment]
Same, very slow
